TACC/Core-Portal
# task/TV3-102: Properly Handle Job Output

## Overview

- Handles terminal job output links for cases where archiving both succeeded and failed, using `remoteOutcome`
- Adds more data to Job History Modal, including a link to the Execution Path for non-terminal jobs
- Readjusts Job Listing column widths
- Increases specificity for the portal name tag in job objects

## Related

* [TV3-102](https://jira.tacc.utexas.edu/browse/TV3-102)



## Testing

1. Run several jobs
2. Go to job history, and confirm Output Path is correctly handled
3. Go to job history, click on a detailed view of a running job, and confirm Execution Path links correctly

## UI
![Screen Shot 2023-01-24 at 3 55 41 PM](https://user-images.githubusercontent.com/20326896/214431614-c2ffb5ea-10c1-4b8d-b80e-e4d96a2bf54b.png)
![Screen Shot 2023-01-24 at 3 49 05 PM](https://user-images.githubusercontent.com/20326896/214431615-fffff5bd-ab92-494e-8398-eb546782c88f.png)
![Screen Shot 2023-01-24 at 3 16 46 PM](https://user-images.githubusercontent.com/20326896/214431621-83f13688-2e9c-458e-a74c-8f6d3bb5aefa.png)
